src/lib/index.ts

```diff
@@ -446,7 +467,7 @@ export class SourceFileConverter {
           if (isDefined) {
             commandStack.pop();
             commands = commandStack[commandStack.length - 1];
-          } else {
+          } else if (!isEmptyElement(node)) {
```

Comment:
Interesting... I'd prefer we output the same text that's in the template, so that if the user wrote `<input></input>`, we'd still write that. We should have more tests around that.

Reply:
I think we'd have to bring in a dom implementation like jsdom to get nextSibling functionality

Reply:
We wouldn't need a DOM implementation - those implement the specific browser-based DOM API  we just need to get the next node in the tree, or the closing tag of the parent. We can do that with the parse5 tree.